packages/sitecore-jss-nextjs/src/editing/editing-render-middleware.test.ts

```diff
@@ -236,8 +236,44 @@ describe('EditingRenderMiddleware', () => {
       expect(res.redirect).to.have.been.calledWith('/custom/path/styleguide');
     });
 
+    it('should handle request when sc_variant and sc_version are not passed', async () => {
+      const queryParams = {
+        mode: 'edit',
+        route: '/styleguide',
+        sc_itemid: '{11111111-1111-1111-1111-111111111111}',
+        sc_lang: 'en',
+        sc_site: 'website',
+        secret: secret,
+      } as MetadataQueryParams;
+
+      const req = mockRequest(EE_BODY, queryParams, 'GET');
+      const res = mockResponse();
+
+      const middleware = new EditingRenderMiddleware();
+      const handler = middleware.getHandler();
+
+      await handler(req, res);
+
+      expect(res.setPreviewData, 'set preview mode w/ data').to.have.been.calledWith({
+        site: 'website',
+        itemId: '{11111111-1111-1111-1111-111111111111}',
+        language: 'en',
+        variantId: '',
+        version: '',
+        editMode: 'metadata',
+        pageState: 'edit',
+      });
+
+      expect(res.redirect).to.have.been.calledOnce;
+      expect(res.redirect).to.have.been.calledWith('/styleguide');
+      expect(res.setHeader).to.have.been.calledWith(
+        'Content-Security-Policy',
+        `frame-ancestors 'self' https://allowed.com ${EDITING_ALLOWED_ORIGINS.join(' ')}`
+      );
+    });
+
     it('should response with 400 for missing query params', async () => {
```

Comment:
I think that this test case is enough, isn't it?
Basically [here](https://github.com/Sitecore/jss/pull/1835/files#diff-12306968a1fa59bd9910f11499c4ebb7ac3f933938fe68e6376aabde1abe52abR289) we shouldn't see sc_version and sc_variant, other parameters should be still there.

Reply:
Yes you're right this test case should be enough.